test/test_integration.py

```diff
@@ -387,7 +387,8 @@ def test_location_link(self):
                 id='test-link-search',
                 href='?testQuery=testValue',
                 refresh=False),
-            html.Button('I am a magic button that updates pathname', id='test-button'),
```

Comment:
We should add `flake8` and `pylint` to the tests, here's how we do it in the `dash` repo:
https://github.com/plotly/dash/blob/25c97d1b3efd99bb7e3f0aa807abb7ebc2712a6b/tox.ini#L17-L18